app/helpers/application_helper.rb

```diff
@@ -61,7 +61,7 @@ def icon_url(name)
   end
 
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
-    return url if Rails.env.development? && (url.blank? || url.exclude?("http"))
+    return url if Rails.env.development? && (url.blank? || url.exclude?("http") || url.include?("fastly"))
```

Comment:
Not very happy with this new condition. However, I believe we want to keep cloudinary option for dev setup and this is the only way I could think of getting this to work.

Reply:
I'm not sure about this, I understand the intention but I feel it should be part of a larger discussion and review about making cloudinary optional or not. If so, we should still probably have a separate PR that does it and a test. Can we table this for now? Maybe open an issue to submit for review

Reply:
I am completely okay with tabling it. The only downside would be that in local dev the site would look like: 

<img width="1531" alt="Screen Shot 2020-03-24 at 5 36 06 PM" src="https://user-images.githubusercontent.com/14208431/77483385-07e43480-6df6-11ea-8606-16139db38691.png">

Reply:
Got it, I asked for help for the review, let's keep it in in the meantime. Thank you!

Reply:
👍

Reply:
Just wanted to follow up on this.